pkg/local_object_storage/metabase/select.go

```diff
@@ -473,6 +536,72 @@ func (db *DB) matchSlowFilters(tx *bbolt.Tx, addr oid.Address, f object.SearchFi
 	}
 
 	for i := range f {
+		switch op := f[i].Operation(); op {
+		case object.MatchNumGT, object.MatchNumGE, object.MatchNumLT, object.MatchNumLE:
```

Comment:
...can help avoid `switch` here (`if op.IsNumeric()`)

Reply:
or local func `if isNumericOp(op)`

Reply:
Can be. Let's do this and then it can be moved to SDK if needed anywhere else.

Reply:
done